FrontEnd/src/Appointment/AppointmentPage.jsx

```diff
@@ -6,34 +6,56 @@ import {
   declineAppointment,
 } from "../BookingPage/BookingDB";
 import "./AppointmentPage.css";
+import { useTheme } from "../UseContext";
 import AppointmentDetails from "./AppointmentDetails";
 import Modal from "../Modal";
 
-function AppointmentPage({ userData }) {
+function AppointmentPage({ userData, socket }) {
   const [isAppointmentDetailsModalShown, setIsAppointmentDetailsModalShown] =
     useState(false);
   const [pendingAppointmentData, setPendingAppointmentData] = useState([]);
   const [upcomingAppointmentData, setUpcomingAppointmentData] = useState([]);
   const userID = userData?.userID;
+  const { theme } = useTheme();
 
   const handleAcceptedAppointment = async (
+    appointment,
     vendorID,
     customerID,
-    appointmentID
+    appointmentID,
+    type
   ) => {
     await acceptAppointment(customerID, vendorID, appointmentID);
-    userAppointmentChanges(customerID);
-    vendorAppointmentChanges(vendorID);
+    await socket.emit("sendNotification", {
+      userID: userData?.userID,
+      senderID: userData?.userID,
+      receiverID: appointment.customerUID,
+      senderName: userData?.UserName,
```

Comment:
Rename `UserName` to `userName` for consistency.

Reply:
It's the name field in my database. It is so that I can differentiate them from form inputs